pageserver/src/import_datadir.rs

```diff
@@ -620,6 +623,68 @@ async fn read_all_bytes(reader: &mut (impl AsyncRead + Unpin)) -> Result<Bytes>
     Ok(Bytes::from(buf))
 }
 
+const YIELD_DIST: u64 = 1024;
```

Comment:
Could move this into should_yield, does not need to be top level, or does something else need this as well?